tests/PhpWord/Shared/HtmlTest.php

```diff
@@ -605,6 +606,21 @@ public function testCouldNotLoadImage()
         Html::addHtml($section, $html, false, true);
     }
 
+    public function testExternalImageSourceNotFound()
+    {
+        $src = 'https://www.bridgewatersavings.com/assets/1442845771-FDIC.png'; // returns 404
+
+        $phpWord = new \PhpOffice\PhpWord\PhpWord();
+        $section = $phpWord->addSection();
+        $html = '<p><img src="' . $src . '" width="150" height="200" style="float: right;"/></p>';
+        Html::addHtml($section, $html, false, true);
+
+        $doc = TestHelperDOCX::getDocument($phpWord, 'Word2007');
+
+        $baseXpath = '/w:document/w:body/w:p/w:r';
+        $this->assertTrue($doc->elementExists($baseXpath . '/w:t'));
```

Comment:
🟢 Could we check if we see the src image instead?

Reply:
sure